src/site/stages/build/page-builder/validator.js

```diff
@@ -0,0 +1,20 @@
+const { Validator } = require('jsonschema');
+
+const {
+  entityReference,
+  entityReferenceArray,
+} = require('./schemas/common/entity-reference');
+const genericNestedString = require('./schemas/common/generic-nested-string');
+
+// Set up formats
+Validator.prototype.customFormats.uuid = input =>
+  /[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}/.test(input);
```

Comment:
Do we need to validate the UUID? Couldn't we just accept it as a string?

Reply:
Right, I was honestly kind of stretching here. Mostly this serves as an example of adding a custom format in case we need it later. I'm totally fine with moving it.

Reply:
Turns out when using `ajv`, because it supports the latest JSON schema draft (07), this isn't needed anymore; `uuid` is a built-in format.